Makefile

```diff
@@ -9,55 +7,34 @@ YAML_FILES+=$(shell find doc/examples -name "cloud-config*.txt" -type f )
 
 PIP_INSTALL := pip install
 
-ifeq ($(PYVER),python3)
-  pyflakes = pyflakes3
-  unittests = unittest3
-  yaml = yaml
-else
-ifeq ($(PYVER),python2)
-  pyflakes = pyflakes
-  unittests = unittest
-else
-  pyflakes = pyflakes pyflakes3
-  unittests = unittest unittest3
-endif
-endif
-
 ifeq ($(distro),)
   distro = redhat
 endif
 
-READ_VERSION=$(shell $(PYVER) $(CWD)/tools/read-version || \
-  echo read-version-failed)
-CODE_VERSION=$(shell $(PYVER) -c "from cloudinit import version; print(version.version_string())")
+READ_VERSION=$(shell python3 $(CWD)/tools/read-version || echo read-version-failed)
+CODE_VERSION=$(shell python3 -c "from cloudinit import version; print(version.version_string())")
```

Comment:
Well, it's not great if the variable is not a constant, otherwise we just end up with a different repeated string (the variable name) and less readable code.

I'm not opposed to have `PYTHON=python3` and then using `$(PYTHON)`, but the change alone won't allow us to really switch interpreter (it's just a Makefile thing), and looks like adding a feature we don't plan to use for the moment.

Reply:
a well named variable is better than a repeated string in the same way that functions are better than copy and pasted code. it isnt' a feature we don't plan to use, its a feature of a programming language that we *are*  using.

Reply:
I still don't fully agree; I will make an extreme example: in a script that calls `/bin/ls` several times, and without the immediate need to test different implementations of `ls`, would you set `LS=ls` and call `$LS` or just call `ls` directly?

Anyway I'm completely fine with having `PYTHON=python3` in this case and I'm adding it right now.